Automattic/jetpack
# WP.com sidebar: fix Tags/Categories order

Fixes p1692176664771369-slack-C029GN3KD

## Proposed changes:


* Updates posts, tags, categories submenus in place rather than mass updating the submenu to avoid getting the tags/categories order skewed. The correct order is `Categories` and then `Tags`

|Before | After|
|-------|------|
|<img width="1715" alt="CleanShot 2023-08-22 at 15 13 43@2x" src="https://github.com/Automattic/jetpack/assets/12430020/914022de-b3bd-4675-a5f9-f58c0bf58bc0">|<img width="1678" alt="CleanShot 2023-08-22 at 15 14 10@2x" src="https://github.com/Automattic/jetpack/assets/12430020/f040f60c-0314-4f08-88b7-f11d601f8fd0">|

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* In your sandbox, run `bin/jetpack-downloader test jetpack fix/nav-unification-post-menu-order` or in an Atomic site use Jetpack Beta and load this branch
* Check that Categories submenu appears before Tags submenu in all cases:
** Go to `https://wordpress.com/settings/taxonomies/category/[DOMAIN]` and Switch to "Classic View"
** Go to `https://wordpress.com/settings/taxonomies/post_tag/[DOMAIN]` and Switch to "Classic View"
** Go to `https://wordpress.com/posts/[DOMAIN]` and Switch to "Classic View"
** Repeat with any combination
